LICENSE

```diff
@@ -1,29 +1,107 @@
-MIT License
-
-Copyright (c) 2018, MIT Data To AI Lab
-
-Copyright (c) 2022 DataCebo, Inc.
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-
-The person using the Software shall indemnify, defend and hold harmless the
-licensors and their respective officers, directors, employees, suppliers,
-successors and assigns from and against any third party claim or demand and any
-associated loss, cost, damages, liability or expense (including reasonable
-attorneys’ fees) to the extent arising out of the use of the Software.
+Business Source License 1.1
+
+Parameters
+
+Licensor:             DataCebo, Inc.
+
+Licensed Work:        RDT
+                      The Licensed Work is (c) DataCebo, Inc.
+
+Additional Use Grant: You may make use of the Licensed Work, and derivatives of the Licensed
+                      Work, provided that you do not use the Licensed Work, or derivatives of
+                      the Licensed Work, for a Data Transform Service.
+
+                      A "Data Transform Service" is a commercial offering
+                      that allows third parties (other than your employees and
+                      contractors) to access the functionality of the Licensed
+                      Work so that such third parties directly benefit from the
+                      reversible data transformation for data formatting, statistical
+                      processing, anonymization or contextual extraction features of
+                      the Licensed Work.
+
+Change Date:          Change date is four years from release date. Please see
+                      https://github.com/sdv-dev/RDT/releases
```

Comment:
Nitpick: Original license take had the sentence "Please see ..." on a new line, right underneath the sentence "Change date is four years ..."